MANIFEST.in

```diff
@@ -1,6 +1,5 @@
 recursive-include chroniker/templates *
 recursive-include chroniker/static *
 recursive-include chroniker/tests/fixtures *
-include requirements-min-django.txt
```

Comment:
Why did you delete this? It's required by the build.

https://travis-ci.org/github/chrisspen/django-chroniker/builds/676288006

Reply:
I merged `requirements-min-django.txt` and `requirements.txt` since I couldn't find a reason why they were in two separate files.

Reply:
It's customary in some Django projects to not explicitly list the Django requirement, since the user will already have Django installed and sometimes the explicit declaration will conflict with the end user's own requirements. However, for testing the build it still needs to be added.